src/Umbraco.Core/Media/EmbedProviders/OEmbedResponse.cs

```diff
@@ -7,6 +7,7 @@ namespace Umbraco.Cms.Core.Media.EmbedProviders
     /// Wrapper class for OEmbed response
     /// </summary>
     [DataContract]
+    // TODO(V10) : change base class to OEmbedProviderBase
```

Comment:
Is this correct? It does not seem to be related to EmbedProvicerBase?

Reply:
No it's not correct indeed. Removed in my latest commit 👍